src/clarity-angular/datagrid/datagrid-cell.ts

```diff
@@ -3,16 +3,56 @@
  * This software is released under MIT license.
  * The full license information can be found in LICENSE in the root directory of this project.
  */
-import {Component} from "@angular/core";
+import { Component } from "@angular/core";
+import { HideableColumnService } from "./providers/hideable-column.service";
 
-@Component({
+@Component ( {
     selector: "clr-dg-cell",
     template: `
         <ng-content></ng-content>
     `,
     host: {
         "[class.datagrid-cell]": "true",
+        "[class.datagrid-cell--hidden]": "hidden"
     }
-})
+} )
 export class DatagridCell {
-}
\ No newline at end of file
+
+    /**
+     * @property _id
+     *
+     * @description
+     * Andentifier for an instance of this cell that maps it to a specific column
+     *
+     * @type {string}
+     */
+    private _id: string;
+
+    /**
+     * @property hidden
+     *
+     * @description
+     * Property used to apply a css class to this cell that hides it when hidden = true.
```

Comment:
Love the use of JSDocs here. I wish we could start doing this throughout our code.